sgpt/utils.py

```diff
@@ -37,6 +39,8 @@ def get_edited_prompt() -> str:
     with NamedTemporaryFile(suffix=".txt", delete=False) as file:
         # Create file and store path.
         file_path = file.name
+        # Write input to file.
+        file.write(input.encode())
```

Comment:
If `prompt` is None then `input` will be `None` and this will break.

```
$ EDITOR=vim sgpt --editor
AttributeError: 'NoneType' object has no attribute 'encode'
```

A simple `if input` wrapper could fix this.